src/rlookup.h

```diff
@@ -441,7 +441,7 @@ void RLookup_AddKeysFrom(const RLookup *src, RLookup *dest, uint32_t flags);
  * Assumes all source keys exist in destination (enforce with ASSERT).
  */
 void RLookupRow_WriteFieldsFrom(const RLookupRow *srcRow, const RLookup *srcLookup,
-                               RLookupRow *destRow, const RLookup *destLookup);
+                               RLookupRow *destRow, RLookup *destLookup);
```

Comment:
Should we comment the assumption that `destLookup` is expected to already have all required keys?
(removing const is just to avoid a warning)